packages/server/src/utils/config.ts

```diff
@@ -0,0 +1,19 @@
+import * as path from 'path'
```

Comment:
as this `config` file is specific to logger, can we move it to `logger.ts`? or is there any specific reason you put it seperately?

Reply:
Yes. The reason for this incompleteness was that I wasn't sure where the project wants to store it's configuration. Could be on package level. Could be on app level. Could be via .env / ENV. As I'm not that experienced with Node projects I didn't know what's the best practice (especially with multiple packages in the same repo)

But if I may suggest something - I would put the configuration in the root directory of the project. Like babel.config.js -> flowise.config.js => but for now it would exceed this PR - so an intermediate solution I created the config.ts within utils -> could also move it one level up to the package root / or leave it out for now. And just mark it for later.

Reply:
since the logger only logs the server, I would leave it within `server` folder. As for where should we put `config.ts`, I don't have a strong preference, we can leave it within `utils` folder for now.

as we make repo more configurable, like database/credentials configuration, we can further improve this file